src/index.html

```diff
@@ -11,6 +11,28 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <label class="label1">
+        <img src="/images/Search.svg" alt="1111" class="img1">
+        <input 
+        id="inp1" 
+        type="text" 
+        class="inp1 inp" 
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="big keypress"
```

Comment:
there should be 2 different forms - so data-qa attributes should be a bit different:
![image](https://user-images.githubusercontent.com/86923089/209951682-0f79a251-c564-4eac-aff9-e3f890f32c58.png)